.rubocop.yml

```diff
@@ -7,22 +7,11 @@ require:
 
 # our custom config
 # TODO: Uncomment it after fix rubocop-todo RSpec/MultipleExpectations
-RSpec/MultipleExpectations:
```

Comment:
I don't think disabling `RSpec/MultipleExpectations` and `RSpec/ExampleLength` makes sense now. We'll get a lot of violations if we do this and it's not optimal to fix them now.